.github/workflows/integration.yml

```diff
@@ -36,6 +36,32 @@ jobs:
         working-directory: ./ops
         run: ./scripts/build-ci.sh
 
+      - name: Save the images
+        working-directory: ./ops
+        run: ./scripts/save-images.sh
```

Comment:
Will this collide with other PRs? ie another PR is submitted that saves images and then retriggering the CI causes different images to be used?

Reply:
I don't think so, each one is in its own unique runner, no?